src/Symfony/Bridge/Twig/Form/TwigRendererEngine.php

```diff
@@ -40,7 +40,7 @@ public function __construct(array $defaultThemes, Environment $environment)
     /**
      * {@inheritdoc}
      */
-    public function renderBlock(FormView $view, $resource, $blockName, array $variables = [])
+    public function renderBlock(FormView $view, $resource, string $blockName, array $variables = [])
```

Comment:
Careful: If you do this, TwigBridge 5.0 will become incompatible with Form 4.4! I would recommend to revert the changes you‘ve made to the bridge.

Reply:
@derrabus , why not bumping up bridge's minimum requirement for form via the `conflict` section?

Reply:
I would not bump the requirements just for having type-hints. We could still do this for 5.1 – there‘s no rush here (imho).